critiquebrainz/frontend/templates/event/entity.html

```diff
@@ -102,6 +102,13 @@ <h4 style="margin-bottom:0;">{{ _('Reviews') }}</h4>
   <div class="col-md-3">
     <h4>{{ _('Event information') }}</h4>
 
+    {% if avg_rating %}
+      <p>
+        <i class="glyphicon glyphicon-star"></i>
+        <span style="font-size: 16px;">{{ avg_rating.rating }}</span><em class="text-muted">/5 based on {{ avg_rating.count }} ratings</em>
+      </p>
+    {% endif %}
```

Comment:
It looks like this part is the same among all templates. It would be a good idea to make it a [macro](http://jinja.pocoo.org/docs/2.9/templates/#macros).